models/lpjguess/R/write.config.LPJGUESS.R

```diff
@@ -183,7 +186,8 @@ write.insfile.LPJGUESS <- function(settings, trait.values, rundir, outdir, run.i
   if (end.year < 1850) {
     CO2 <- data.frame(start.year:end.year, rep(280, n.year))
   } else if (end.year < 2021) {
-    data(co2.1850.2020, package = "PEcAn.LPJGUESS")
+    utils::data(co2.1850.2020, package = "PEcAn.LPJGUESS", envir = co2_data)
```

Comment:
@harshagr70 unless I missed where it's defined above, I think you need to define `co2_data` as a new environment before assigning into it:
```suggestion
    co2_data <- new.env()
    utils::data(co2.1850.2020, package = "PEcAn.LPJGUESS", envir = co2_data)
```

Reply:
@infotroph i defined it on line 16 , but i guess it would be much better to define it just before we assign it

Reply:
So you did! Yeah, I think since this is effectively a temporary variable that we write to once and immediately read back out to `co2.1850.2020`, defining it just before use is clearer.